packages/runtime/container-runtime/src/summarizerTypes.ts

```diff
@@ -132,6 +133,43 @@ export type GenerateSummaryResult =
     | IUploadSummaryResult
     | ISubmitSummaryOpResult;
 
+export interface IBroadcastSummaryResult {
+    readonly summarizeOp: ISummaryOpMessage;
+    readonly broadcastDuration: number;
+}
+
+export interface IAckNackSummaryResult {
+    readonly summaryAckNackOp: ISummaryAckMessage | ISummaryNackMessage;
+    readonly ackNackDuration: number;
+}
+
+export type SummarizeResultPart<T> = {
+    success: true;
+    data: T;
+} | {
+    success: false;
+    data: T | undefined;
+    message: string;
+    error: any;
+};
+
+export interface ISummarizeResult {
+    /** Resolves when we generate, upload, and submit the summary */
+    readonly generateSummary: Promise<SummarizeResultPart<GenerateSummaryResult>>;
+    /** Resolves when we see our summarize op broadcast; is sequence number of op */
+    readonly broadcastSummaryOp: Promise<SummarizeResultPart<IBroadcastSummaryResult>>;
+    /** True for ack; false for nack */
```

Comment:
What is true for ack and false for nack here?

Reply:
Also outdated comment. Will remove.